.github/workflows/ci.yml

```diff
@@ -76,7 +76,7 @@ jobs:
     runs-on: ubuntu-16.04
     if: startsWith(github.ref, 'refs/tags/v')
     env:
-      MIX_ENV: dev
+      MIX_ENV: docs
```

Comment:
ended up allowing the `test/support` directory to be compiled in `Mix.env() == :dev` so that you can try out scripting against the `SlipstreamWeb.Endpoint` locally

but I don't want all the `@moduledoc`d stuff in `test/` to show up in docs, so I made a new `MIX_ENV` just for docs

I think there's prior art out there that I've seen do this, don't have a link handy